test/types/real/nan-minmax.chpl

```diff
@@ -42,15 +42,15 @@ testNans();
 // also add +- infinity
 ARR[1+1] = plusinf;
 ARR[n-1] = minusinf;
-assert(min reduce ARR == -INFINITY);
-assert(max reduce ARR ==  INFINITY);
-assert(minloc reduce zip(ARR, ARR.domain) == (-INFINITY, n-1));
-assert(maxloc reduce zip(ARR, ARR.domain) == ( INFINITY, 1+1));
+assert(min reduce ARR == -inf);
+assert(max reduce ARR ==  inf);
+assert(minloc reduce zip(ARR, ARR.domain) == (-inf, n-1));
+assert(maxloc reduce zip(ARR, ARR.domain) == ( inf, 1+1));
 testNans();
 
 writeln("done");
 
-// min/max reductions on ARR with 1 or 2 NANs added to it
+// min/max reductions on ARR with 1 or 2 NaNs added to it
```

Comment:
This isn't as important as the first two since it's not chapel docs, and also because "nans" is weird to read. But I would change it for consistency